src/main/java/UseCases/ReadableFromFile.java

```diff
@@ -0,0 +1,9 @@
+package UseCases;
+
+import java.io.IOException;
+
+public interface ReadableFromFile {
+
+    Object readFromFile(String filepath) throws IOException, ClassNotFoundException;
+
+}
```

Comment:
I like what you did here Rudy, it seems to adhere well to the SOLID principles, by using more specific interfaces rather than implementing unused methods. Also, great job refactoring, as all other actors still communicate with readwriter the exact same!